packages/webpack/src/runtime/runtime.js

```diff
@@ -0,0 +1,189 @@
+// @ts-check
+/// <reference path="./runtime.d.ts" />
+/* global LAVAMOAT */
+/* global lockdown, harden, Compartment */
+
+const {
+  create,
+  freeze,
+  assign,
+  defineProperty,
+  defineProperties,
+  getOwnPropertyDescriptors,
+} = Object
+const warn = typeof console === 'object' ? console.warn : () => {}
+// Avoid running any wrapped code or using compartment if lockdown was not called.
+// This is for when the bundle ends up running despite SES being missing.
+// It was previously useful for sub-compilations running an incomplete bundle as part of the build, but currently that is being skipped. We might go back to it for the sake of build time security if it's deemed worthwihile in absence of lockdown.
+const LOCKDOWN_ON = typeof lockdown !== 'undefined'
+if (LOCKDOWN_ON) {
+  lockdown(LAVAMOAT.options.lockdown)
+} else {
+  warn(
+    'LavaMoat: runtime execution started without SES present, switching to no-op.',
+  )
+}
+
+const { getEndowmentsForConfig, copyWrappedGlobals } =
+  LAVAMOAT.endowmentsToolkit()
+
+// These must match assumptions in the wrapper.js
+// sharedKeys are included in the runtime
+
+const { NAME_globalThis, NAME_scopeTerminator, NAME_runtimeHandler } =
```

Comment:
what is the significance of these variable names?

Reply:
discussed in person